samtranslator/schema_source/aws_serverless_statemachine.py

```diff
@@ -4,8 +4,8 @@
 
 from typing_extensions import Literal
 
-from schema_source.aws_serverless_connector import EmbeddedConnector
-from schema_source.common import (
+from samtranslator.schema_source.aws_serverless_connector import EmbeddedConnector
+from samtranslator.schema_source.common import (
     BaseModel,
     DictStrAny,
     PassThroughProp,
```

Comment:
1. `cloudformation-docs.json` should stay in `schema_source`.
2. New files should be under `internal` so we don't increase our 'public' interface.

Reply:
good suggestion. udpated